src/TextArea/TextArea.tsx

```diff
@@ -10,11 +10,11 @@ interface TextAreaProps extends InputProps {
   width?: string
 }
 
-export const PlainTextArea: React.FC<TextAreaProps & OmitOnChange> = ({ onChange, ...rest }) => (
-  <textarea onChange={(event) => onChange(event.target.value, event)} {...rest}/>
+export const TextArea: React.FC<TextAreaProps & OmitOnChange> = ({ onChange, ...rest }) => (
+  <StyledTextArea onChange={(event) => onChange(event.target.value, event)} {...rest}/>
 )
 
-export const TextArea = styled(PlainTextArea)`
+export const StyledTextArea = styled('textarea') <{ width?: string }>`
```

Comment:
nit: I'd re-order this file so that `StyledTextArea` is defined before it's used in `TextArea`, confused me for a sec